src/components/RadioGroup/RadioGroup.spec.jsx

```diff
@@ -0,0 +1,81 @@
+import _ from 'lodash';
+import React from 'react';
+import assert from 'assert';
+// import sinon from 'sinon';
+import { mount, shallow } from 'enzyme';
+
+import getRandom from '../../util/random';
+import { common } from '../../util/generic-tests';
+
+import RadioGroup from './RadioGroup';
+
+describe('RadioGroup', () => {
+	common(RadioGroup);
+
+	describe('props', () => {
+		describe('name', () => {
+			it('sets the `name` attribute of the child radio buttons.', () => {
+				const name = getRandom();
+				const wrapper = shallow(
+					<RadioGroup name={name}>
+						<RadioGroup.RadioButton />
+						<RadioGroup.RadioButton />
+						<RadioGroup.RadioButton />
+					</RadioGroup>
+				);
+
+				_.forEach(wrapper.find('RadioButton').nodes, (node) => {
+					assert.equal(node.props.name, name);
+				});
+			});
+
+			it('defaults to a randomly generated string that is passed along to the children.', () => {
```

Comment:
the test text can leave out the random part since it's not checking for randomness.